.codecov.yml

```diff
@@ -1,11 +1,19 @@
 codecov:
+  branch: dev
```

Comment:
This means that we are going to test the coverage only when we have a PR to dev?

Reply:
No. This means that the `dev` branch is used for the main configuration. Some options can be loaded right on the branch of a PR, but others need a "global" default that will use that branch.